### DISCUSSION TITLE
Anhtraquic horizon

### DISCUSSION THREAD

**Discussion by StephanMantelISRIC** [iuss-wrb/wrb#2] [Diagnostics]
Dear all

I have some problems with the anthraquic properties of the surface horizon of paddy field soil profiles in Cambodia

The top  soil is very light even the C content is higher tha 3%, lighter than the requested colour
The bulk density below the ploughpan, which is not obvious because the top soil is very sandy, is not allways 10% higher than the puddle horizon.
(Bulk density with 100 cc cylinders and 3 replicates)

Some photos and soils characteristics of 4 profiles of the Chrey Bak watershed in the centre en Cambodia.
Pascal Podwojewski[
[Anthraquic WRB.pdf](https://github.com/iuss-wrb/wrb/files/13239197/Anthraquic.WRB.pdf)
](url)

**Comment by StephanMantelISRIC** [comment]
Dear Pascal,
thank you for your remarks from Cambodian soils. Of course, I added them to the list of suggestions for the next version of WRB.
But before we change the criteria, we must answer the question, whether the Anthraquic must be recognised on each rice field?
It means, we must discuss whether the particular colours and the clear difference in bulk density are essential for soils with Anthraquic or not? 
These are many important questions which require further discussion. I think we should collect more opinions from the people experienced in this topic, or even arrange the WRB workshop in the area with such soils.

Cezary Kabala

**Comment by StephanMantelISRIC** [comment]
Dear Cezary
The problem is that in the same watershed, the colour of the surface horizon of most of soil profiles does not complete the requests for atnraquic. It is much lighter and sometimes more pinkish (5YR)
For the bulk density it is more complex. There is a sharp differenciation between the claric surface Apr horizon and the gleyic features below (a lot o spots grey and orange, very classical).
Generally a compacted layer occurs below this A horizon (sometimes >1.9 g/cm3) but NOT ALWAYS while the soils are morphologically not very different. It means if we do not measure systematically the bulk density we cannot guess if there is a strong compaction or not.

The variability of theses soils is the grainsize content, from loamy to very sandy depending the location in the watershed

FOR YOUR INFORMATION
Our research work is to measure the impact of termite mounts (over 2m high -fungii growers  with comb chambers ) in this environment.
If the soil is sandy, the termite mound is more more loamy (enriched in fine particles)
If the soil is loamy there is no significant difference
But in any cases the termite mounds have a higher pH and contain much more Calcium (Calcium oxalate generated by termites) which is used as amendment in these soils generally with acid pH.

Pascal

**Comment by StephanMantelISRIC** [comment]
Dear Pascal,

thank you very much for your detailed information. Generally, I agree with Cezary that we may accept paddy soils without anthraquic (and hydragric) horizon. 

The criteria have been first written by Otto Spaargaren for WRB 2006, based on the Chinese Soil Taxonomy. The Chinese ST does not know gleyic properties. For that reason, the colour criteria came in (criterion 1 and 2.c). Maybe, we can replace them by simply requiring gleyic properties. Nevertheless, I am surprised by these high Munsell values in the presence of so much Corg. Do you have an explanation for that? 

I think, we do need a criterion for a dense plough pan. For the moment, it is relative. But we may add an absolute criterion as a second option.

See the attached draft for a new short anthraquic horizon.
[Anthraquic horizon_Peter Schad.docx](https://github.com/iuss-wrb/wrb/files/13239230/Anthraquic.horizon_Peter.Schad.docx)

Peter Schad

**Comment by StephanMantelISRIC** [comment]
Dear all, Thank you for your explanations
I will check again the colour, we have collected samples for further analysis they are in a storage room in the Institute of Technology of Cambodia which is our partnership and the partnership of Gembloux (Belgium). Same for Soil organic carbon content. I will confirm you.
Also the surprise was than in soils with 90% sand, we found many earthworms. Generally they don't like sandy soils.
I will go to Cambodia this Thursday. We will sample end of November, the rice fields are still too wet yet.
Pascal

**Comment by StephanMantelISRIC** [comment]
Dear Peter,
referring to your proposal of a new wording of Anthraquic horizon, I fully agree that we may add alternative density criterium for plough pan and gleyic properties instead of particular colours. However, there is an important difference between this new proposal and older requirements, because the old definition required features of gleyic properties in both parts of Anthraquic horizon (puddled and plough pan), while the new one requires gleyic properties anywhere in the horizon, e.g. only in a plough pan. Is it intentional resignation from the gleyic requirements for the puddled layer? Moreover, the density 1.6 seems low. The second (alternative) criterion shouldn't be much weaker than the basic criterion, or maybe even clearly stronger, to be sure that improper soils will not meet this requirement...
Of course, we hope that the criteria will be applied only in rice fields, but who knows? I'm sure you remember the troubles with pretic, when people started to recognise it in each archeological pit:) 
I will try to collect more analytical and field data.
Kind regards
Cezary

**Comment by StephanMantelISRIC** [comment]
Freddy Nachtergaele: Many typical paddy soils do not meet the strict chroma requirement of Anthraquic. The first requirement for the Anthraquic horizon is to have a chroma <=2 and this criteria is not fulfilled in 4 monoliths now classified as Anthrosols reviewed in the soil reference collection of ISRIC (https://wsm.isric.org/gallery.html). It is suggested to review the criterion with Asian colleagues.

**Comment by mukundgofast99** [comment]
Your observations about the anthraquic properties in Cambodian paddy fields are quite interesting. The lighter topsoil, despite a higher C content, could be due to its sandy nature, affecting bulk density and color perception. Have you considered how organic pest control methods might influence soil properties over time? In India, [MM Pest Control](mmpestcontrol.in) specializes in organic solutions that help maintain soil health while effectively managing pests. It would be interesting to see how natural pest control measures could impact bulk density and soil structure in paddy fields. Looking forward to seeing the soil profile data!
